test/test_torch.py

```diff
@@ -16703,6 +16705,15 @@ def test_remainder_fmod_large_dividend(self, device, dtype):
                 else:
                     self.assertTrue(abs(c[0] - d[0]) == abs(b[0]))  # differ by one divisor
 
+    @dtypes(torch.float32, torch.float64)
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    def test_hypot(self, device, dtype):
+        t1 = torch.tensor([3.0, 0, 4.5, -5.5], dtype=dtype, device=device)
```

Comment:
This is a fine test but you might as well use randn to generate a set of tensors and then compare them with NumPy.

For example:

```
inputs = [
  (torch.randn((3, 3,), device=device, dtype=dtype), torch.randn((3, 3,), device=device, dtype=dtype),
  ...
]
```

You could include a 1D set of inputs and a 3D set of inputs, for example, plus a set of inputs that are transposed.

Reply:
Better also add a test case where the tensor on the lhs in a long tensor, and the tensor on the rhs is a tensor of the specified dtype.